.env.example

```diff
@@ -132,3 +132,9 @@ EMAIL_SERVER_PASSWORD='<office365_password>'
 
 # Set the following value to true if you wish to enable Team Impersonation
 NEXT_PUBLIC_TEAM_IMPERSONATION=false
+
+# Close.com internal CRM
+CLOSECOM_API_KEY=
+
+# Sendgrid internal email sender
+SENDGRID_API_KEY=
```

Comment:
These are the ones I mentioned @zomars, only for internal use and available for any self-hoster who wants to track down key info from their users in Sendgrid and Close.com